src/sections/Hero/Hero.tsx

```diff
@@ -25,46 +25,27 @@ const ButtonsWrapperOffset = styled.div`
 
   ${media.between('md', 'lg')`
     gap: ${rem('10px')} ${rem('5px')}; 
-`}
+  `}
 
   ${media.between('lg', 'xl')`
     gap: ${rem('10px')} ${rem('12px')};
   `}
 
-  ${media.lessThan('sm')`
+  @media (max-width: 427px) {
```

Comment:
https://github.com/vazco/forminer/assets/42475735/b5b2c6e6-f02b-4ad9-8fca-aa7bd4348ce7




There's a width where the buttons act unnaturally. Once there are next to each other, then they are in the column aligned to the left, and then to the center. Probably we should skip the part with the left alignment.